src/analyze/analyze.c

```diff
@@ -92,7 +92,7 @@ static usec_t arg_base_time = USEC_INFINITY;
 STATIC_DESTRUCTOR_REGISTER(arg_dot_from_patterns, strv_freep);
 STATIC_DESTRUCTOR_REGISTER(arg_dot_to_patterns, strv_freep);
 
-struct boot_times {
+typedef struct BootTimes {
```

Comment:
once upon a time we had this rule that noone followed that structs specific to a source file should use "struct" and no "typedef" and be lower_case and stuff exported to other .c files should use typedefs and user CamelCase. But i guess it's a mess right now, and the rule might not be the smartest rule to have, hence looks ok to me

Reply:
Oh, OK. Anyway, I don't think this is even remotely true. We should probably convert the remaining few instances without typedef too.